lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandler.java

```diff
@@ -129,9 +129,6 @@ public APIGatewayProxyResponseEvent handleRequest(
             LogHelper.attachGovukSigninJourneyIdToLogs(
                     clientSessionDetailsDto.getGovukSigninJourneyId());
 
-            userIdentityService.deleteUserIssuedCredentialsIfAnyExpired(
```

Comment:
Should be able to remove `userIdentityService` from here completely I think? (and template stuff)

Reply:
Good spot - yes, can be removed completely!